admin/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "OSF_Admin",
+  "version": "0.0.0",
+  "description": "Facilitating Open Science",
+  "repository": "https://github.com/CenterForOpenScience/osf.io",
+  "author": "Center for Open Science",
+  "dependencies": {
+    "URIjs": "^1.14.1",
+    "assets-webpack-plugin": "^0.1.0",
+    "async": "^0.9.0",
+    "babel-loader": "^4.1.0",
+    "body-parser": "~1.12.0",
+    "bootbox": "^4.4.0",
+    "bower": "^1.3.12",
+    "c3": "^0.4.10",
+    "css-loader": "^0.9.1",
+    "diff": "~1.2.2",
+    "dropzone": "git://github.com/sloria/dropzone.git#accept-directory",
+    "exports-loader": "^0.6.2",
+    "express": "~4.10.0",
+    "file-loader": "^0.8.1",
+    "font-awesome-webpack": "0.0.3",
+    "gsap": "^1.17.0",
+    "highlight.js": "~8.4.0",
+    "historyjs": "^1.8.0-b2",
+    "html-loader": "^0.2.3",
+    "imports-loader": "^0.6.3",
+    "jquery.cookie": "^1.4.1",
+    "json-loader": "^0.5.1",
+    "jstimezonedetect": "^1.0.5",
+    "knockout": "~3.2.0",
+    "knockout.punches": "^0.5.1",
+    "knockout.validation": "^2.0.2",
+    "livedb": "~0.4.8",
+    "livedb-mongo": "~0.4.1",
+    "markdown-it": "~4.1.0",
+    "markdown-it-ins-del": "^0.1.1",
+    "markdown-it-sanitizer": "~0.3.0",
+    "markdown-it-toc": "~1.1.0",
+    "markdown-it-video": "~0.2.1",
+    "mime-types": "~2.0.12",
+    "mithril": "^0.2",
+    "moment": "^2.9.0",
+    "morgan": "^1.5.1",
+    "object-assign": "^2.0.0",
+    "pikaday": "^1.3.2",
+    "pretty-data": "^0.40.0",
+    "pym.js": "^0.4.1",
+    "raven": "^0.7.2",
+    "raw-loader": "^0.5.1",
+    "share": "0.7.27",
+    "style-loader": "^0.8.3",
+    "treebeard": "git://github.com/caneruguz/treebeard.git#9f1458a9f5961460dcd84c833b5da0180219f253",
+    "typeahead.js": "^0.10.5",
+    "url-loader": "^0.5.5",
+    "webpack": "^1.7.2",
+    "webpack-bundle-tracker": "0.0.8",
+    "ws": "~0.4.32"
+  },
+  "devDependencies": {
+    "chai": "^2.1.1",
+    "faker": "^2.1.2",
+    "jshint": "2.8.0",
+    "karma": "^0.12.31",
+    "karma-chrome-launcher": "^0.1.7",
+    "karma-firefox-launcher": "^0.1.4",
+    "karma-mocha": "^0.1.10",
+    "karma-phantomjs-launcher": "^0.1.4",
+    "karma-sauce-launcher": "^0.2.10",
+    "karma-sinon": "^1.0.4",
+    "karma-sourcemap-loader": "^0.3.4",
+    "karma-spec-reporter": "0.0.16",
+    "karma-webpack": "~1.5.0",
+    "mocha": "^2.1.0"
```

Comment:
If you're going to keep your package.json (I'd suggest not because keeping these 60+ versions in sync with OSF won't be fun) I'd add some run commands to do webpack and run js tests and install bower packages, etc. Either here or in the invoke scripts.